apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -0,0 +1,35 @@
+<ng-container [ngTemplateOutlet]="cardWrapper"></ng-container>
+<ng-template #cardWrapper>
+  <div
+    class="flex w-fit flex-col gap-3 rounded-md border-2 border-black p-4"
+    [class]="customClass()">
+    <ng-container
+      [ngTemplateOutlet]="
+        headerTemplate?.templateRef || defaultHeaderTemplate
+      "></ng-container>
+    <ng-template #defaultHeaderTemplate>
+      This is the default card header
+    </ng-template>
+
+    <section>
+      @for (item of list(); track item.id) {
+        <app-list-item [item]="item" (onDeleteItem)="deleteItem($event)">
+          <ng-template #itemContentTemplate>
+            <div>{{ item[prop()] }}</div>
+          </ng-template>
+        </app-list-item>
```

Comment:
you made things a bit complicated and you coupled listItem with card, So you cannot use something else inside your card. But it's working.

Reply:
I get your point, I will refactor and decouple the card component. Thanks for your instant feedback.